ocaml/parsing/pprintast.ml

```diff
@@ -1809,8 +1810,11 @@ and directive_argument f x =
   | Pdir_ident (li) -> pp f "@ %a" longident li
   | Pdir_bool (b) -> pp f "@ %s" (string_of_bool b)
 
-and jane_syntax_expr ctxt f (jexp : Jane_syntax.Expression.t) =
-  match jexp with
+and jane_syntax_expr ctxt attrs f (jexp : Jane_syntax.Expression.t) =
+  if attrs <> [] then
+    pp f "((%a)@,%a)" (jane_syntax_expr ctxt []) jexp
+      (attributes ctxt) attrs
```

Comment:
It makes me sad that we have to duplicate this but I don't see a good way around it, do you?

Reply:
I don't either.

Reply:
Maybe just a comment noting that, then?

Reply:
I weakly push back against this. I think, in this case, the comment does not have much interesting to say (the code duplication is clear enough, but it's simple and I think obvious as to what it's doing).

Reply:
This view is not strongly held, so I acquiesce happily